packages/bathymetry/src/stac.ts

```diff
@@ -0,0 +1,64 @@
+import { Tile, TileMatrixSet } from '@basemaps/geo';
+import { LogType, ProjectionTileMatrixSet } from '@basemaps/shared';
+import * as cp from 'child_process';
+import * as path from 'path';
+import { BathyMaker } from './bathy.maker';
+import { FileType } from './file';
+import { Hash } from './hash';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const packageJson = require('../package.json');
+
+function getCommitHash(): string {
+    return cp.execSync('git rev-parse HEAD').toString().trim();
+}
+
+/** Write some basic metadata about how the file was created*/
+async function create(bm: BathyMaker, tile: Tile, logger: LogType): Promise<Record<string, unknown>> {
+    const tileId = TileMatrixSet.tileToName(tile);
+    const outputTiffPath = bm.file.name(FileType.Output, tileId);
+    const ptms = ProjectionTileMatrixSet.get(bm.tms.projection.code);
+
+    const bounds = ptms.tileToWgs84Bounds(tile);
+    const geometry = bounds.toPolygon();
```

Comment:
I think this needs a bit more `geojson` around it; guessing: `{type: 'Polygon', coordinates: bounds.toPolygon()[0]}`